sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowFunctionFrame.scala

```diff
@@ -157,6 +157,38 @@ final class OffsetWindowFunctionFrame(
   override def currentUpperBound(): Int = throw new UnsupportedOperationException()
 }
 
+class FixedOffsetWindowFunctionFrame(
```

Comment:
Add documentation.

I would like to see a unit test for this.

Reply:
I want reference the unit test for `OffsetWindowFunctionFrame`, can you tell me where they are?